### ISSUE TITLE
In-game group/company

### ISSUE THREAD

**Issue by Sydwan** [StrangeLoopGames/EcoSuggestions#1724]
Hello, after some servers and many struggles, I would suggest devs to add a kind of in-game group/company where you can add your trusted friends/teammates in :
- having your friends already authorized on any claim/deed/vehicle you place
- sharing storages connections and orders (so you don't have to bully your mate cause he did not link the right storage :) )

This group/company would be higher in the auth-checking process so it ignores current system of authorizations you manually give on claim/deed/vehicle, even if current one has to remain for temporary auth on your claim.

Adding it as an in-game feature (law item/demographic) could also open new horizons in terms of laws drafting.

Sydwan

**Comment by iuriguilherme** [comment]
I believe you can currently do this with a combination of an elected title and a demographic

**Comment by Sydwan** [comment]
> I believe you can currently do this with a combination of an elected title and a demographic

I know you can create title and add them into it (demographic are more about laws stuff) but you still need to ad that tittle EVERY TIME you create a deed, especially for vehicles thats is really annoying, cause whenever you log off without authing your friends on the skid for example, they are f up :/
Basically, that would be a title where you add once your friends into it, then every time you create a new claim, place a new vehicle etc, the game auto auth them on it

**Comment by iuriguilherme** [comment]
Sounds like your feature request would be to save a vehicle's deed owner in the vehicle data if it's a title and use that when the new deed is created instead of the player who placed the vehicle

**Comment by Sydwan** [comment]
> Sounds like your feature request would be to save a vehicle's deed owner in the vehicle data if it's a title and use that when the new deed is created instead of the player who placed the vehicle

Exactly, same could go for claims (even if claims are way less annoying since you only claim/unclaim times to times).
